rpm/move-profiled-config.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+OLD_DIR="$HOME/.profiled"
+NEW_DIR="$HOME/.config/profiled"
+
+mkdir -p "$NEW_DIR"
+if [ -d "$OLD_DIR" ] ; then
+  mv -f "$OLD_DIR/*" "$NEW_DIR"
```

Comment:
When is this executed? Are there any guarantees that profiled is not running? If not, then this is not a hazard free way to do migration. I would have simply loaded from legacy dir and new dir and then removed the legacy dir from the daemon itself -> no chances of timing hazards.

Reply:
I agree best to do it there.